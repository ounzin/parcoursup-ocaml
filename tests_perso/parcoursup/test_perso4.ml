open Test_parcoursup_utils2
let session = nouvelle_session ()

let () =  
  ajoute_candidat session ~nom_candidat:"Ahmed";
  ajoute_candidat session ~nom_candidat:"Houefa";
  ajoute_formation session ~nom_formation:"IUT Lokossa" ~capacite:1;
  ajoute_formation session ~nom_formation:"Université Grenoble Alpes" ~capacite:1;
  ajoute_voeu session 
    ~rang_repondeur:(None) 
    ~nom_candidat:"Ahmed" 
    ~nom_formation:"IUT Lokossa";
    ajoute_voeu session 
    ~rang_repondeur:(None)
    ~nom_candidat:"Ahmed" 
    ~nom_formation:"Université Grenoble Alpes";
    ajoute_voeu session 
    ~rang_repondeur:(None)
    ~nom_candidat:"Houefa" 
    ~nom_formation:"IUT Lokossa";
    ajoute_voeu session 
    ~rang_repondeur:(None) 
    ~nom_candidat:"Houefa" 
    ~nom_formation:"Université Grenoble Alpes";
  ajoute_commission session 
    ~nom_formation:"IUT Lokossa" 
    ["Houefa";"Ahmed"]; 
    ajoute_commission session 
    ~nom_formation:"Université Grenoble Alpes" 
    ["Houefa";"Ahmed"]; 
  reunit_commissions session;
  nouveau_jour session;
  affiche_voeux_en_attente session "Ahmed";
  affiche_propositions_en_attente session "Ahmed";
  affiche_voeux_en_attente session "Houefa";
  affiche_propositions_en_attente session "Houefa";
  renonce session  ~nom_candidat:"Houefa"  ~nom_formation:"IUT Lokossa";
  nouveau_jour session;
  affiche_voeux_en_attente session "Ahmed";
  affiche_propositions_en_attente session "Ahmed";
  affiche_voeux_en_attente session "Houefa";
  affiche_propositions_en_attente session "Houefa";