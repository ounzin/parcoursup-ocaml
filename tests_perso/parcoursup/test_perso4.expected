nouvelle session
ajoute candidat Ahmed
ajoute candidat Houefa
ajoute formation IUT Lokossa (1 places)
ajoute formation Université Grenoble Alpes (1 places)
ajoute voeu de Ahmed pour IUT Lokossa
ajoute voeu de Ahmed pour Université Grenoble Alpes
ajoute voeu de Houefa pour IUT Lokossa
ajoute voeu de Houefa pour Université Grenoble Alpes
commission de classement de IUT Lokossa : [Houefa; Ahmed]
commission de classement de Université Grenoble Alpes : [Houefa; Ahmed]
réunit commissions
nouveau jour
Voeux en attente de Ahmed : {IUT Lokossa(0), Université Grenoble Alpes(0)}
Propositions en attente de Ahmed : {}
Voeux en attente de Houefa : {}
Propositions en attente de Houefa : {IUT Lokossa, Université Grenoble Alpes}
Houefa renonce à IUT Lokossa
nouveau jour
Voeux en attente de Ahmed : {Université Grenoble Alpes(0)}
Propositions en attente de Ahmed : {IUT Lokossa}
Voeux en attente de Houefa : {}
Propositions en attente de Houefa : {Université Grenoble Alpes}
